### ISSUE TITLE
No Party Option on the Paperdoll

### ISSUE THREAD

**Issue by greeduomacro** [ZaneDubya/UltimaXNA#421]
You cannot party with other players due to the lack of a party manifest and nothing happening from typing /party. See below for how it should be:

![image](https://cloud.githubusercontent.com/assets/8658771/13552198/40d82c6a-e30d-11e5-92bf-d69abaf85af4.png)

**Comment by msx752** [comment]
Look at https://github.com/ZaneDubya/UltimaXNA/pull/400 , https://github.com/ZaneDubya/UltimaXNA/issues/330

**Comment by ZaneDubya** [comment]
Note to self: double check remaining packets:

```
Packet 0xBF/0x06: Party System

Subsubcommand 1: Add a party member (4 bytes)
BYTE[4] id (if 0, a targeting cursor appears)Client Message

Subsubcommand 1: Add party member(s) (1+ numMembers*4)
BYTE[1] number of Members (total number of members in the party)
Then, for each member in numMembers:
BYTE[4] id
Server Message

Subsubcommand 2: Remove a party member (4 bytes)
BYTE[4] id (if 0, a targeting cursor appears)
Client message

Subsubcommand 2: Remove a party member (? Bytes)
BYTE[1] number of Members (total number of members in the new party)
BYTE[4] idofPlayerRemoved
Then, for each member in numMembers:
BYTE[4] id
Server message

Subsubcommand 3: Tell party member a message (Variable # of bytes)
BYTE[4] id (of target, from client, of source, from server)
BYTE[n][2] Null terminated Unicode message.
Client & Server Message

Subsubcommand 4: Tell full party a message (Variable # of bytes)
BYTE[n][2] Null terminated Unicode message.
Client Message.

Subsubcommand 4: Tell full party a message (Variable # of bytes)
BYTE[4] id (of source)
BYTE[n][2] Null terminated Unicode message.
Server Message

Subsubcommand 6: Party Can Loot Me? (1 byte)
BYTE[1] canloot (0=no, 1=yes)
Client message

Subsubcommand 7: party invitation?
BYTE[4] serial (party leader)

Subsubcommand 8: Accept join party invitation (4 bytes)
BYTE[4] serial (party leader)
Client message

Subsubcommand 9: Decline join party invitation (4 bytes)
BYTE[4] serial (party leader)
Client message
```

**Comment by ZaneDubya** [comment]
Merged in #451. Please open a new issue for any bugs/feature additions with the party system.
